recipe/thirdpartyemailpassword/index.ts

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2020, VRAI Labs and/or its affiliates. All rights reserved.
```

Comment:
Please make sure all copyright has 2021. You can do a global find / replace.

Reply:
done